### PR TITLE
Integrate hello-swap as btc-eth example in create-comit-app

### PR DESCRIPTION
**Pull request by da-kami**
* move code from hello-swap into new_project/examples/btc-eth
* add build script that compresses new_project folder into archive
* adapt new command to uncompress the new_project content

### CODE REVIEW

File: new_project/examples/btc-eth/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Hello-Swap
```
Comment (D4nte) [inline new_project/examples/btc-eth/README.md]: The full README should go to the root of the `new_project` folder.
Here, should be a one liner saying that in this example the maker buy X ETH for Y BTC from the taker.

File: new_project/examples/btc-eth/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
+    "name": "hello_swap",
+    "version": "1.0.0",
+    "description": "A Hello World COMIT app that does a swap",
```
Comment (D4nte) [inline new_project/examples/btc-eth/package.json]:
```suggestion
    "description": "A Hello World COMIT example app that does a Bitcoin-Ether swap",
```

File: new_project/examples/btc-eth/package.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{
+    "name": "hello_swap",
```
Comment (D4nte) [inline new_project/examples/btc-eth/package.json]:
```suggestion
    "name": "btc_eth_hello_swap",
```

File: new_project/package.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "new_project",
```
Comment (D4nte) [inline new_project/package.json]:
```suggestion
  "name": "new_comit_app",
```

File: Cargo.toml [hunk 5]
```diff
@@ -30,3 +30,10 @@ toml = "0.5"
 unzip = "0.1"
 ureq = { version = "0.11", features = ["rustls"] }
```
Comment (D4nte) [inline Cargo.toml]: This can go away

File: new_project/package.json (hunk 4 above)
Comment (thomaseizinger) [reply new_project/package.json]: We should actually adapt this based on the users input!

Comment (D4nte) [reply new_project/package.json]: Yep, I forget to comment on this suggestion, I just wanted something a bit less generic. Adapting on user input sounds good to me.

Comment (thomaseizinger) [reply new_project/package.json]: Will make a ticket if you don't mind! :)

Comment (thomaseizinger) [reply new_project/package.json]: https://github.com/comit-network/create-comit-app/issues/92

**Pull request merged**